neqo-transport/src/connection.rs

```diff
@@ -1339,8 +1398,14 @@ impl Connection {
             };
 
             let header_start = encoder.len();
-            let (pt, pn, mut builder) =
-                Self::build_packet_header(path, *space, encoder, tx, &self.retry_info);
+            let (pt, pn, mut builder) = Self::build_packet_header(
```

Comment:
cool, I was thinking the same thing.

Reply:
ok, trying thing runs into borrowck issues on `self` in `output_path` since `self.crypto.states` is already mutably borrowed. Going to leave as-is for now.

Reply:
Ahh, I remember now why I made it like this in the first place.  Damn you borrow checker.